### PR TITLE
Update Changelog with last few patch releases

### PR DESCRIPTION
**Pull request by kaushikb11**
## What does this PR do?

Update Changelog with last few patch releases

## Before submitting
- [x] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [ ] Did you make sure to update the documentation with your changes? (if necessary)
- [ ] Did you write any new necessary tests? (not for typos and docs)
- [ ] Did you verify new and existing tests pass locally with your changes?
- [x] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Review by ananthsub — APPROVED** [review]

**Review by Borda — APPROVED** [review]
Pls add also 1.2.9 and 1.2.10 to legacy testing

**Review by awaelchli — APPROVED** [review]

**Review by s-rog — APPROVED** [review]

**Comment by kaushikb11** [comment]
@Borda How should I add legacy checkpoints to S3?

**Comment by Borda** [comment]
> @Borda How should I add legacy checkpoints to S3?

you need to pull the existing, the follow https://github.com/PyTorchLightning/pytorch-lightning/tree/master/legacy
and later upload the legacy back, you shall have access to the our AWS :]

**Review by tchaton — APPROVED** [review]
LGTM !

File: tests/checkpointing/test_legacy_checkpoints.py [hunk 1]
```diff
@@ -60,6 +60,9 @@
         "1.2.5",
         "1.2.6",
         "1.2.7",
+        "1.2.8",
+        "1.2.9",
+        "1.2.10",
```
Comment (kaushikb11) [inline tests/checkpointing/test_legacy_checkpoints.py:65]:
```suggestion
        "1.2.8",
        "1.2.10",
```

**Pull request closed without merging**